libc/include/llvm-libc-macros/features-macros.h

```diff
@@ -0,0 +1,15 @@
+//===-- Definition of macros from features.h
+//----------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __LLVM_LIBC_MACROS_FEATURES_MACROS_H
+#define __LLVM_LIBC_MACROS_FEATURES_MACROS_H
+
+#define __LLVMLIBC__ 1
```

Comment:
Why `__LLVMLIBC__` instead of `__LLVM_LIBC__`? Is there some prior art here?

Reply:
I just picked one, also wondered whether `LLVM_LIBC` or `LLVMLIBC` fit better here. Could change it to __LLVM_LIBC__ if you prefer

Reply:
Sounds good, merged with `__LLVM_LIBC__`